### DISCUSSION TITLE
Will Zag power ChakraUI someday?

### DISCUSSION THREAD

**Discussion by ivanjeremic** [chakra-ui/zag#1824] [Q&A] [answered]
Is it planned for Zag.js to power ChakraUI someday? If yes will this be in some major release?

**Comment by segunadebayo** [comment, marked as answer]
Yes. Zag is powering the next major release of Chakra (v3)

It will be a major change of course but we’re excited about this.

**Reply by ivanjeremic** [reply]
Very exciting, I was looking at all three projects (chakra-ui, panda-CSS, zag.js). I was not sure how to look at those projects, and now I think I have an idea that you may already be considering If PandaCSS in the CSS styling engine and Zag.js is a headless & styleless component library Chakra could be just a theme for Zag (one of many to come if wished). 

The flow would be Zag components > install Chakra theme. What I want to say is that Chakra's future could be becoming just a theme for Zag.

**Reply by segunadebayo** [reply]
That’s exactly how we see it as well. You’re on point!
